### DISCUSSION TITLE
无法播放音乐，专辑页面不显示专辑内容

### DISCUSSION THREAD

**Discussion by skyoko** [qier222/YesPlayMusic#1180] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/96154771/148029949-cd39a5f9-cc68-44b5-b8a7-3540890e1776.png)

### 反馈 bug 需要的信息

- 客户端 0.4.3
- win10

**Comment by skyoko** [comment]
MV可以播放

**Comment by memorydream** [comment]
无法复现，建议尝试重启

**Comment by skyoko** [comment]
重新安装也不可以，之前是可以用的

**Comment by skyoko** [comment]
在网页端尝试也是这个情况
![image](https://user-images.githubusercontent.com/96154771/148140235-21947e35-7a8f-48a9-8dfa-e23bade98acf.png)

**Comment by ZhuliuAiagle** [comment]
我这边也有同样的问题，能加载出来歌单封面但是加载不出来歌曲
![image](https://user-images.githubusercontent.com/29112148/148150875-3c599ff2-4a0c-4b98-9b95-78538d26c557.png)

**Comment by nafuchan** [comment]
我一直停止工作，之前版本还能用

**Comment by pan93412** [comment, marked as answer]
> 我这边也有同样的问题，能加载出来歌单封面但是加载不出来歌曲
> ![image](https://user-images.githubusercontent.com/29112148/148150875-3c599ff2-4a0c-4b98-9b95-78538d26c557.png)

試試看重新登入？如果重新登入還是不行，請照著以下步驟重設 YPM。

https://user-images.githubusercontent.com/28441561/148158459-e0e0de32-2dff-468e-9621-6f9414a4c7c1.mov

**Reply by pan93412** [reply]
<img width="1440" alt="截圖 2022-01-05 上午11 59 13" src="https://user-images.githubusercontent.com/28441561/148158622-2416f7ec-d3fc-48d1-b7d6-e3db93001b9b.png">

網頁端也正常。你也可以照著以下影片重設 resetApp。

https://user-images.githubusercontent.com/28441561/148158758-c4391789-352c-4503-ab68-ef180b6b4907.mov

**Reply by ZhuliuAiagle** [reply]
您好，执行重设操作之后可以正常加载歌曲列表了。感谢！
